hack/tools/catalogs/lib/bundle.sh

```diff
@@ -0,0 +1,191 @@
+# Library of functions for interacting with bundles and their manifests
+
+# SCRIPT_ROOT is the root of the script
+source "$(dirname "${BASH_SOURCE[0]}")/hash.sh"
+
+# Given package and version grabs the bundle image from stdin FBC stream
+function get-bundle-image(){
```

Comment:
Should we document anywhere that this is used, or maybe as part of this utility function first do a check for the presence, that `jq` is required to be installed?